### DISCUSSION TITLE
Compatibility with other models

### DISCUSSION THREAD

**Discussion by spider876** [aredden/electron-G14Control#107] [Ideas]
I can confirm this program works 100% with asus scar strix 2021 G533QS.

**Comment by frmariam** [comment]
TUF FX505DY laptop
It allows me to select which Armoury Crate plan (in fan curve editor) even after I removed it (it degraded my laptop performance and was causing my wifi card to randomly stop being recognized in Win until next reboot... fixed after removal).
Fn+F5 doesn't switch plans.

gives me a "failed to set fan curve" with green checkmark... but actually changes fan plan.
fails to apply g14Control plans
